bin/qds.py

```diff
@@ -524,6 +542,10 @@ def main():
                          default=False,
                          help="skip verification of server SSL certificate. Insecure: use with caution.")
 
+    optparser.add_option("--cloud", dest="cloud",
+                         default=os.getenv('CLOUD_PROVIDER'),
+                         help="cloud", choices=["AWS", "AZURE", "ORACLE_BMC"])
```

Comment:
Yes I am doing that only. For api_token and api_url we use env variables as default which is same as above for cloud too :).